### ISSUE TITLE
Does not pull views from Tableau

### ISSUE THREAD

**Issue by matthewreily** [tableau/custom-portal-sample#2]
Installed and configured the application as described in the readme file and am unable to pull any views in from Tableau.

I noticed that there are a few issues in the master branch.
1. There is a left over merge comment in selfserviceanalytics.html
2. The application could not resolve the reference to scl-alpha.js
3. The application could not resolve the reference to tableau-server-client.js

**Comment by benlower** [comment]
@matthewreily thanks for posting this. will take a look to see what is wrong.

**Comment by benlower** [comment]
@matthewreily Thanks again for reporting these issues. I've resolved the issues you reported:

1. Cleaned these up. Looks like there was a conflict that we missed when we renamed 'tableau-server-client.js' to 'scl-alpha.js'.
2. added 'scl-alpha.js' to the webpack config file.
3. removed references to 'tableau-server-client.js'

I also added 'ArtistFinder.twbx' to the repo. This workbook needs to be published to your server if you are going to use the New Artist Finder page and the d3 network diagram.

Let us know if you run into any other issues.
